qiskit/synthesis/__init__.py

```diff
@@ -162,4 +163,8 @@
 from .unitary import aqc
 from .one_qubit import OneQubitEulerDecomposer
 from .two_qubit.xx_decompose import XXDecomposer
-from .two_qubit.two_qubit_decompose import TwoQubitBasisDecomposer, two_qubit_cnot_decompose
+from .two_qubit.two_qubit_decompose import (
+    TwoQubitBasisDecomposer,
+    two_qubit_cnot_decompose,
```

Comment:
I think this `two_qubit_cnot_decompose` is missing an `.. autofunction::` in the docstring?  If it's meant to be there, let's take a note to add it after rc1 - no need to hold up the PR over a documentation thing.

Reply:
Ah, yeah good catch I missed that in the original review. I'll open an issue to track this. It actually will likely require a full docstring addition as it's doesn't have anything there now: https://github.com/Qiskit/qiskit/blob/main/qiskit/synthesis/two_qubit/two_qubit_decompose.py#L1553 

it's a lazy loaded instance of `TwoQubitBasisDecomposer` using cx.

Reply:
I actually already opened #11709 for it - forgot to comment here.

Reply:
I also wasn't sure what to do with this function, since it has no docstring, and also due to the comment here:
https://github.com/Qiskit/qiskit/blob/a234ff2dc717884fa9b1c3dce3e3d8c4c3eccbfb/qiskit/synthesis/two_qubit/two_qubit_decompose.py#L1531

Reply:
Thanks for taking care of this PR